PipetteCommandRaw command
---
bool success
uint32 err
---
float32 completion   # TODO %
bool moving
